### PR TITLE
ERMS-2311 Umfrage Bugs

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp [hunk 1]
```diff
@@ -452,6 +343,118 @@
...
+                                                number="${costItemSub?.costInBillingCurrency}"
+                                                minFractionDigits="2" maxFractionDigits="2"
+                                                type="number"/></b>
+
+                                        ${(costItemSub?.billingCurrency?.getI10n('value').split('-')).first()}
+
+                                        <g:set var="oldCostItem"
+                                               value="${costItemSub.costInBillingCurrency ?: 0.0}"/>
+
+                                        <g:if test="${costItemSub?.startDate || costItemSub?.endDate}">
```
Comment (klodav) [inline app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: to be or not to be -> ?

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp [hunk 2]
```diff
@@ -452,6 +343,118 @@
...
+                            </th>
+                            <th>Diff.</th>
+                        </tr>
+                        </thead>
+                        <tbody class="top aligned">
+                        <tr>
+                            <td>
+                                <g:if test="${costItemsSub}">
+                                    <g:each in="${costItemsSub}" var="costItemSub">
+                                        ${costItemSub?.costItemElement?.getI10n('value')}
```
Comment (klodav) [inline app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: to be or not to be -> ?

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp [hunk 3]
```diff
@@ -452,6 +343,118 @@
...
+                                        <b><g:formatNumber
+                                                number="${costItemSub?.costInBillingCurrency}"
```
Comment (klodav) [inline app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: to be or not to be -> ?

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp [hunk 4]
```diff
@@ -452,6 +343,118 @@
...
+                                            <br>(${formatDate(date: costItemSub?.startDate, format: message(code: 'default.date.format.notime'))} - ${formatDate(date: costItemSub?.endDate, format: message(code: 'default.date.format.notime'))})
+                                        </g:if>
+                                        <br>
+
+                                    </g:each>
+                                </g:if>
+                            </td>
+                            <td>
+                                <g:if test="${costItemSurvey}">
+                                    ${costItemSurvey?.costItemElement?.getI10n('value')}
```
Comment (klodav) [inline app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: to be or not to be -> ?

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp (hunk 2 above)
Comment (croqueGrec09) [reply app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: Aggree

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp (hunk 3 above)
Comment (croqueGrec09) [reply app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: Agree

**Pull request merged**